## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 424008`
Commit: `9e1ca46eaa27dc1c97f477d163f83c5750ca1bcb`

---
### `x86_64-linux`
<details>
  <summary>:x: 234 packages failed to build:</summary>
  <ul>
    <li>aider-chat (python312Packages.aider-chat)</li>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-bedrock</li>
    <li>aider-chat-with-bedrock.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>aider-chat-with-playwright</li>
    <li>aider-chat-with-playwright.dist</li>
    <li>aider-chat.dist (python312Packages.aider-chat.dist)</li>
    <li>autotools-language-server</li>
    <li>autotools-language-server.dist</li>
    <li>bagels</li>
    <li>bagels.dist</li>
    <li>browsr</li>
    <li>browsr.dist</li>
    <li>dooit</li>
    <li>dooit-extras</li>
    <li>dooit-extras.dist</li>
    <li>dooit.dist</li>
    <li>frogmouth</li>
    <li>frogmouth.dist</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>isd</li>
    <li>isd.dist</li>
    <li>isponsorblocktv</li>
    <li>isponsorblocktv.dist</li>
    <li>memray</li>
    <li>memray.dist</li>
    <li>mutmut</li>
    <li>mutmut.dist</li>
    <li>newelle</li>
    <li>oterm</li>
    <li>oterm.dist</li>
    <li>posting</li>
    <li>posting.dist</li>
    <li>private-gpt</li>
    <li>private-gpt.dist</li>
    <li>python312Packages.llama-cloud-services</li>
    <li>python312Packages.llama-cloud-services.dist</li>
    <li>python312Packages.llama-index</li>
    <li>python312Packages.llama-index-agent-openai</li>
    <li>python312Packages.llama-index-agent-openai.dist</li>
    <li>python312Packages.llama-index-cli</li>
    <li>python312Packages.llama-index-cli.dist</li>
    <li>python312Packages.llama-index-core</li>
    <li>python312Packages.llama-index-core.dist</li>
    <li>python312Packages.llama-index-embeddings-gemini</li>
    <li>python312Packages.llama-index-embeddings-gemini.dist</li>
    <li>python312Packages.llama-index-embeddings-google</li>
    <li>python312Packages.llama-index-embeddings-google.dist</li>
    <li>python312Packages.llama-index-embeddings-huggingface</li>
    <li>python312Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python312Packages.llama-index-embeddings-ollama</li>
    <li>python312Packages.llama-index-embeddings-ollama.dist</li>
    <li>python312Packages.llama-index-embeddings-openai</li>
    <li>python312Packages.llama-index-embeddings-openai.dist</li>
    <li>python312Packages.llama-index-graph-stores-nebula</li>
    <li>python312Packages.llama-index-graph-stores-nebula.dist</li>
    <li>python312Packages.llama-index-graph-stores-neo4j</li>
    <li>python312Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python312Packages.llama-index-graph-stores-neptune</li>
    <li>python312Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python312Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python312Packages.llama-index-legacy</li>
    <li>python312Packages.llama-index-legacy.dist</li>
    <li>python312Packages.llama-index-llms-ollama</li>
    <li>python312Packages.llama-index-llms-ollama.dist</li>
    <li>python312Packages.llama-index-llms-openai</li>
    <li>python312Packages.llama-index-llms-openai-like</li>
    <li>python312Packages.llama-index-llms-openai-like.dist</li>
    <li>python312Packages.llama-index-llms-openai.dist</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai</li>
    <li>python312Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python312Packages.llama-index-program-openai</li>
    <li>python312Packages.llama-index-program-openai.dist</li>
    <li>python312Packages.llama-index-question-gen-openai</li>
    <li>python312Packages.llama-index-question-gen-openai.dist</li>
    <li>python312Packages.llama-index-readers-database</li>
    <li>python312Packages.llama-index-readers-database.dist</li>
    <li>python312Packages.llama-index-readers-file</li>
    <li>python312Packages.llama-index-readers-file.dist</li>
    <li>python312Packages.llama-index-readers-json</li>
    <li>python312Packages.llama-index-readers-json.dist</li>
    <li>python312Packages.llama-index-readers-llama-parse</li>
    <li>python312Packages.llama-index-readers-llama-parse.dist</li>
    <li>python312Packages.llama-index-readers-s3</li>
    <li>python312Packages.llama-index-readers-s3.dist</li>
    <li>python312Packages.llama-index-readers-twitter</li>
    <li>python312Packages.llama-index-readers-twitter.dist</li>
    <li>python312Packages.llama-index-readers-txtai</li>
    <li>python312Packages.llama-index-readers-txtai.dist</li>
    <li>python312Packages.llama-index-readers-weather</li>
    <li>python312Packages.llama-index-readers-weather.dist</li>
    <li>python312Packages.llama-index-vector-stores-chroma</li>
    <li>python312Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python312Packages.llama-index-vector-stores-google</li>
    <li>python312Packages.llama-index-vector-stores-google.dist</li>
    <li>python312Packages.llama-index-vector-stores-postgres</li>
    <li>python312Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python312Packages.llama-index-vector-stores-qdrant</li>
    <li>python312Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python312Packages.llama-index.dist</li>
    <li>python312Packages.llama-parse</li>
    <li>python312Packages.llama-parse.dist</li>
    <li>python312Packages.manifestoo</li>
    <li>python312Packages.manifestoo.dist</li>
    <li>python312Packages.private-gpt</li>
    <li>python312Packages.private-gpt.dist</li>
    <li>python312Packages.pytest-textual-snapshot</li>
    <li>python312Packages.pytest-textual-snapshot.dist</li>
    <li>python312Packages.textual</li>
    <li>python312Packages.textual-autocomplete</li>
    <li>python312Packages.textual-autocomplete.dist</li>
    <li>python312Packages.textual-dev</li>
    <li>python312Packages.textual-dev.dist</li>
    <li>python312Packages.textual-fastdatatable</li>
    <li>python312Packages.textual-fastdatatable.dist</li>
    <li>python312Packages.textual-serve</li>
    <li>python312Packages.textual-serve.dist</li>
    <li>python312Packages.textual-slider</li>
    <li>python312Packages.textual-slider.dist</li>
    <li>python312Packages.textual-textarea</li>
    <li>python312Packages.textual-textarea.dist</li>
    <li>python312Packages.textual-universal-directorytree</li>
    <li>python312Packages.textual-universal-directorytree.dist</li>
    <li>python312Packages.textual.dist</li>
    <li>python313Packages.llama-cloud-services</li>
    <li>python313Packages.llama-cloud-services.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-agent-openai</li>
    <li>python313Packages.llama-index-agent-openai.dist</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-gemini</li>
    <li>python313Packages.llama-index-embeddings-gemini.dist</li>
    <li>python313Packages.llama-index-embeddings-google</li>
    <li>python313Packages.llama-index-embeddings-google.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud</li>
    <li>python313Packages.llama-index-indices-managed-llama-cloud.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai</li>
    <li>python313Packages.llama-index-multi-modal-llms-openai.dist</li>
    <li>python313Packages.llama-index-program-openai</li>
    <li>python313Packages.llama-index-program-openai.dist</li>
    <li>python313Packages.llama-index-question-gen-openai</li>
    <li>python313Packages.llama-index-question-gen-openai.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-llama-parse</li>
    <li>python313Packages.llama-index-readers-llama-parse.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-postgres</li>
    <li>python313Packages.llama-index-vector-stores-postgres.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.llama-parse</li>
    <li>python313Packages.llama-parse.dist</li>
    <li>python313Packages.manifestoo</li>
    <li>python313Packages.manifestoo.dist</li>
    <li>python313Packages.private-gpt</li>
    <li>python313Packages.private-gpt.dist</li>
    <li>python313Packages.pytest-textual-snapshot</li>
    <li>python313Packages.pytest-textual-snapshot.dist</li>
    <li>python313Packages.textual</li>
    <li>python313Packages.textual-autocomplete</li>
    <li>python313Packages.textual-autocomplete.dist</li>
    <li>python313Packages.textual-dev</li>
    <li>python313Packages.textual-dev.dist</li>
    <li>python313Packages.textual-fastdatatable</li>
    <li>python313Packages.textual-fastdatatable.dist</li>
    <li>python313Packages.textual-serve</li>
    <li>python313Packages.textual-serve.dist</li>
    <li>python313Packages.textual-slider</li>
    <li>python313Packages.textual-slider.dist</li>
    <li>python313Packages.textual-textarea</li>
    <li>python313Packages.textual-textarea.dist</li>
    <li>python313Packages.textual-universal-directorytree</li>
    <li>python313Packages.textual-universal-directorytree.dist</li>
    <li>python313Packages.textual.dist</li>
    <li>rexi</li>
    <li>rexi.dist</li>
    <li>rich-cli</li>
    <li>rich-cli.dist</li>
    <li>smassh</li>
    <li>smassh.dist</li>
    <li>streamcontroller</li>
    <li>tftui</li>
    <li>tftui.dist</li>
    <li>toolong</li>
    <li>toolong.dist</li>
    <li>upiano</li>
    <li>upiano.dist</li>
    <li>wsrepl</li>
    <li>wsrepl.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 535 packages built:</summary>
  <ul>
    <li>codecov-cli</li>
    <li>codecov-cli.dist</li>
    <li>keymap-drawer (python313Packages.keymap-drawer)</li>
    <li>keymap-drawer.dist (python313Packages.keymap-drawer.dist)</li>
    <li>python312Packages.grep-ast</li>
    <li>python312Packages.grep-ast.dist</li>
    <li>python312Packages.keymap-drawer</li>
    <li>python312Packages.keymap-drawer.dist</li>
    <li>python312Packages.lsp-tree-sitter</li>
    <li>python312Packages.lsp-tree-sitter.dist</li>
    <li>python312Packages.tree-sitter</li>
    <li>python312Packages.tree-sitter-c-sharp</li>
    <li>python312Packages.tree-sitter-c-sharp.dist</li>
    <li>python312Packages.tree-sitter-embedded-template</li>
    <li>python312Packages.tree-sitter-embedded-template.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-bash</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-bash.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-beancount</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-beancount.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-bibtex</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-bibtex.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-bitbake</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-bitbake.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-bqn</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-bqn.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-c</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-c-sharp</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-c-sharp.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-c.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-clojure</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-clojure.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-cmake</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-cmake.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-comment</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-comment.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-commonlisp</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-commonlisp.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-cpp</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-cpp.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-crystal</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-crystal.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-css</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-css.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-cuda</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-cuda.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-cue</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-cue.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-dart</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-dart.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-devicetree</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-devicetree.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-dockerfile</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-dockerfile.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-dot</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-dot.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-earthfile</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-earthfile.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-eex</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-eex.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-elisp</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-elisp.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-elixir</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-elixir.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-elm</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-elm.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-embedded-template</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-embedded-template.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-erlang</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-erlang.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-factor</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-factor.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-fennel</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-fennel.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-fish</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-fish.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-fortran</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-fortran.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-gdscript</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-gdscript.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-gemini</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-gemini.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-gleam</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-gleam.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-glimmer</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-glimmer.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-glsl</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-glsl.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-go</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-go.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-godot-resource</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-godot-resource.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-gomod</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-gomod.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-gowork</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-gowork.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-graphql</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-graphql.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-haskell</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-haskell.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-hcl</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-hcl.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-heex</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-heex.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-hjson</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-hjson.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-html</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-html.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-http</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-http.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-hyprlang</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-hyprlang.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-janet-simple</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-janet-simple.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-java</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-java.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-javascript</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-javascript.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-jsdoc</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-jsdoc.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-json</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-json.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-json5</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-json5.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-jsonnet</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-jsonnet.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-julia</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-julia.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-just</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-just.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-kdl</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-kdl.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-koka</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-koka.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-kotlin</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-kotlin.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-latex</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-latex.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ledger</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ledger.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-llvm</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-llvm.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-lua</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-lua.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-make</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-make.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-markdown</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-markdown-inline</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-markdown-inline.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-markdown.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-netlinx</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-netlinx.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-nickel</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-nickel.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-nix</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-nix.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-norg</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-norg-meta</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-norg-meta.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-norg.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-nu</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-nu.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ocaml</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ocaml-interface</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ocaml-interface.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ocaml.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-org-nvim</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-org-nvim.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-perl</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-perl.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-pgn</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-pgn.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-php</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-php.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-pioasm</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-pioasm.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-prisma</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-prisma.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-proto</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-proto.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-pug</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-pug.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-python</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-python.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ql</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ql-dbscheme</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ql-dbscheme.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ql.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-query</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-query.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-r</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-r.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-regex</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-regex.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-rego</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-rego.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-river</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-river.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-rst</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-rst.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ruby</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-ruby.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-rust</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-rust.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-scala</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-scala.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-scheme</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-scheme.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-scss</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-scss.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-smithy</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-smithy.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-solidity</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-solidity.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-sparql</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-sparql.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-supercollider</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-supercollider.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-surface</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-surface.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-svelte</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-svelte.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-talon</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-talon.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-tera</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-tera.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-tiger</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-tiger.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-tlaplus</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-tlaplus.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-toml</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-toml.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-tsq</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-tsq.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-tsx</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-tsx.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-turtle</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-turtle.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-twig</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-twig.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-typescript</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-typescript.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-typst</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-typst.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-uiua</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-uiua.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-verilog</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-verilog.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-vim</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-vim.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-vue</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-vue.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-wgsl</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-wgsl.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-wing</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-wing.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-yaml</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-yaml.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-yang</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-yang.dist</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-zig</li>
    <li>python312Packages.tree-sitter-grammars.tree-sitter-zig.dist</li>
    <li>python312Packages.tree-sitter-language-pack</li>
    <li>python312Packages.tree-sitter-language-pack.dist</li>
    <li>python312Packages.tree-sitter-markdown</li>
    <li>python312Packages.tree-sitter-markdown.dist</li>
    <li>python312Packages.tree-sitter-yaml</li>
    <li>python312Packages.tree-sitter-yaml.dist</li>
    <li>python312Packages.tree-sitter.dist</li>
    <li>python313Packages.grep-ast</li>
    <li>python313Packages.grep-ast.dist</li>
    <li>python313Packages.lsp-tree-sitter</li>
    <li>python313Packages.lsp-tree-sitter.dist</li>
    <li>python313Packages.tree-sitter</li>
    <li>python313Packages.tree-sitter-c-sharp</li>
    <li>python313Packages.tree-sitter-c-sharp.dist</li>
    <li>python313Packages.tree-sitter-embedded-template</li>
    <li>python313Packages.tree-sitter-embedded-template.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-bash</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-bash.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-beancount</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-beancount.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-bibtex</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-bibtex.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-bitbake</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-bitbake.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-bqn</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-bqn.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-c</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-c-sharp</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-c-sharp.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-c.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-clojure</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-clojure.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cmake</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cmake.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-comment</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-comment.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-commonlisp</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-commonlisp.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cpp</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cpp.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-crystal</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-crystal.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-css</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-css.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cuda</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cuda.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cue</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-cue.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dart</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dart.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-devicetree</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-devicetree.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dockerfile</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dockerfile.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dot</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-dot.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-earthfile</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-earthfile.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-eex</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-eex.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-elisp</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-elisp.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-elixir</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-elixir.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-elm</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-elm.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-embedded-template</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-embedded-template.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-erlang</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-erlang.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-factor</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-factor.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fennel</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fennel.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fish</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fish.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fortran</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-fortran.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-gdscript</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-gdscript.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-gemini</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-gemini.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-gleam</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-gleam.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-glimmer</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-glimmer.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-glsl</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-glsl.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-go</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-go.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-godot-resource</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-godot-resource.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-gomod</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-gomod.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-gowork</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-gowork.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-graphql</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-graphql.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-haskell</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-haskell.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-hcl</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-hcl.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-heex</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-heex.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-hjson</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-hjson.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-html</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-html.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-http</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-http.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-hyprlang</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-hyprlang.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-janet-simple</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-janet-simple.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-java</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-java.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-javascript</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-javascript.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-jsdoc</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-jsdoc.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-json</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-json.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-json5</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-json5.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-jsonnet</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-jsonnet.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-julia</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-julia.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-just</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-just.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-kdl</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-kdl.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-koka</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-koka.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-kotlin</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-kotlin.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-latex</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-latex.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ledger</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ledger.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-llvm</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-llvm.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-lua</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-lua.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-make</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-make.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-markdown</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-markdown-inline</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-markdown-inline.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-markdown.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-netlinx</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-netlinx.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nickel</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nickel.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nix</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nix.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-norg</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-norg-meta</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-norg-meta.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-norg.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nu</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-nu.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ocaml</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ocaml-interface</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ocaml-interface.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ocaml.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-org-nvim</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-org-nvim.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-perl</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-perl.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-pgn</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-pgn.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-php</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-php.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-pioasm</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-pioasm.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-prisma</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-prisma.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-proto</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-proto.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-pug</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-pug.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-python</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-python.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ql</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ql-dbscheme</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ql-dbscheme.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ql.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-query</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-query.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-r</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-r.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-regex</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-regex.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rego</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rego.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-river</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-river.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rst</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rst.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ruby</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-ruby.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rust</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-rust.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-scala</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-scala.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-scheme</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-scheme.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-scss</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-scss.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-smithy</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-smithy.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-solidity</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-solidity.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-sparql</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-sparql.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-supercollider</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-supercollider.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-surface</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-surface.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-svelte</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-svelte.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-talon</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-talon.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-tera</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-tera.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-tiger</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-tiger.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-tlaplus</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-tlaplus.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-toml</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-toml.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-tsq</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-tsq.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-tsx</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-tsx.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-turtle</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-turtle.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-twig</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-twig.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typescript</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typescript.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typst</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-typst.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-uiua</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-uiua.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-verilog</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-verilog.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vim</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vim.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vue</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-vue.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-wgsl</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-wgsl.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-wing</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-wing.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-yaml</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-yaml.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-yang</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-yang.dist</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-zig</li>
    <li>python313Packages.tree-sitter-grammars.tree-sitter-zig.dist</li>
    <li>python313Packages.tree-sitter-language-pack</li>
    <li>python313Packages.tree-sitter-language-pack.dist</li>
    <li>python313Packages.tree-sitter-markdown</li>
    <li>python313Packages.tree-sitter-markdown.dist</li>
    <li>python313Packages.tree-sitter-yaml</li>
    <li>python313Packages.tree-sitter-yaml.dist</li>
    <li>python313Packages.tree-sitter.dist</li>
    <li>textlsp</li>
    <li>textlsp.dist</li>
    <li>vectorcode</li>
    <li>vectorcode.dist</li>
    <li>vimPlugins.vectorcode-nvim</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>autotools-language-server</summary>
<pre>        """
        tree = self.prepare(uri, tree, reset)
>       captures = self.query.captures(tree.root_node)
E       AttributeError: 'tree_sitter.Query' object has no attribute 'captures'

/nix/store/a2yc06z8klwd2racdj452i9fgs2xhj6w-python3.13-lsp-tree-sitter-0.0.17/lib/python3.13/site-packages/lsp_tree_sitter/finders.py:598: AttributeError
=============================== warnings summary ===============================
../../nix/store/a2yc06z8klwd2racdj452i9fgs2xhj6w-python3.13-lsp-tree-sitter-0.0.17/lib/python3.13/site-packages/lsp_tree_sitter/finders.py:11
../../nix/store/a2yc06z8klwd2racdj452i9fgs2xhj6w-python3.13-lsp-tree-sitter-0.0.17/lib/python3.13/site-packages/lsp_tree_sitter/finders.py:11
  /nix/store/a2yc06z8klwd2racdj452i9fgs2xhj6w-python3.13-lsp-tree-sitter-0.0.17/lib/python3.13/site-packages/lsp_tree_sitter/finders.py:11: DeprecationWarning: Importing Validator directly from the jsonschema package is deprecated and will become an ImportError. Import it from jsonschema.protocols instead.
    from jsonschema import Validator

tests/test_utils.py::Test::test_check
  /nix/store/fjblw292r37h541glj0dnhbwj43nzr94-autotools-language-server-0.0.22/lib/python3.13/site-packages/make_language_server/utils.py:39: DeprecationWarning: query() is deprecated. Use the Query() constructor instead.
    QUERIES[name] = parser.language.query(text)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_utils.py::Test::test_check - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
=================== 1 failed, 2 passed, 3 warnings in 3.87s ====================</pre>
</details>
<details>
<summary>python312Packages.textual</summary>
<pre>tests/text_area/test_languages.py::test_update_highlight_query
tests/text_area/test_setting_themes.py::test_default_theme
tests/text_area/test_setting_themes.py::test_setting_builtin_themes
tests/text_area/test_setting_themes.py::test_setting_unknown_theme_raises_exception
tests/text_area/test_setting_themes.py::test_registering_and_setting_theme
  /nix/store/xczjsi1jv2l5mnhipcfli0lqavzqwwsn-python3.12-textual-3.7.0/lib/python3.12/site-packages/textual/document/_syntax_aware_document.py:63: DeprecationWarning: query() is deprecated. Use the Query() constructor instead.
    return self.language.query(query)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_widget.py::test_widget_construct - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_languages.py::test_register_language - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_languages.py::test_setting_unknown_language - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_languages.py::test_setting_language_to_none - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_languages.py::test_update_highlight_query - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_setting_themes.py::test_default_theme - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_setting_themes.py::test_setting_builtin_themes - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_setting_themes.py::test_setting_unknown_theme_raises_exception - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_setting_themes.py::test_registering_and_setting_theme - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
= 9 failed, 2840 passed, 1 skipped, 4 xfailed, 10 warnings in 124.59s (0:02:04) =</pre>
</details>
<details>
<summary>python313Packages.textual</summary>
<pre>tests/text_area/test_languages.py::test_update_highlight_query
tests/text_area/test_setting_themes.py::test_registering_and_setting_theme
tests/text_area/test_setting_themes.py::test_default_theme
tests/text_area/test_setting_themes.py::test_setting_unknown_theme_raises_exception
tests/text_area/test_setting_themes.py::test_setting_builtin_themes
  /nix/store/xxikpswrmbcazvfr1lm2ajslxh0244px-python3.13-textual-3.7.0/lib/python3.13/site-packages/textual/document/_syntax_aware_document.py:63: DeprecationWarning: query() is deprecated. Use the Query() constructor instead.
    return self.language.query(query)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_widget.py::test_widget_construct - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_languages.py::test_register_language - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_languages.py::test_setting_unknown_language - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_languages.py::test_setting_language_to_none - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_languages.py::test_update_highlight_query - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_setting_themes.py::test_registering_and_setting_theme - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_setting_themes.py::test_default_theme - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_setting_themes.py::test_setting_unknown_theme_raises_exception - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
FAILED tests/text_area/test_setting_themes.py::test_setting_builtin_themes - AttributeError: 'tree_sitter.Query' object has no attribute 'captures'
= 9 failed, 2840 passed, 1 skipped, 4 xfailed, 10 warnings in 74.70s (0:01:14) =</pre>
</details>
</details>

--- a/pkgs/development/python-modules/lsp-tree-sitter/default.nix
+++ b/pkgs/development/python-modules/lsp-tree-sitter/default.nix
@@ -45,5 +45,7 @@ buildPythonPackage rec {
     homepage = "https://github.com/neomutt/lsp-tree-sitter";
     license = licenses.gpl3Only;
     maintainers = with maintainers; [ doronbehar ];
+    # https://github.com/neomutt/lsp-tree-sitter/issues/4
+    broken = true;
   };
 }

--- a/pkgs/development/python-modules/textual/default.nix
+++ b/pkgs/development/python-modules/textual/default.nix
@@ -101,5 +101,7 @@ buildPythonPackage rec {
     changelog = "https://github.com/Textualize/textual/blob/${src.tag}/CHANGELOG.md";
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [ gepbird ];
+    # https://github.com/Textualize/textual/issues/5868
+    broken = true;
   };
 }

--- a/pkgs/development/python-modules/tree-sitter/default.nix
+++ b/pkgs/development/python-modules/tree-sitter/default.nix
@@ -15,7 +15,7 @@
 
 buildPythonPackage rec {
   pname = "tree-sitter";
-  version = "0.24.0";
+  version = "0.24.0-unstable-2025-06-02";
   pyproject = true;
 
   disabled = pythonOlder "3.10";
@@ -23,8 +23,8 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "tree-sitter";
     repo = "py-tree-sitter";
-    tag = "v${version}";
-    hash = "sha256-ZDt/8suteaAjGdk71l8eej7jDkkVpVDBIZS63SA8tsU=";
+    rev = "9c78f3b8d10f81b97fbb2181c9333323d6375480";
+    hash = "sha256-jPqTraGrYFXBlci4Zaleyp/NTQhvuI39tYWRckjnV2E=";
     fetchSubmodules = true;
   };
 
@@ -56,11 +56,21 @@ buildPythonPackage rec {
     "test_dot_graphs"
   ];
 
-  meta = {
-    description = "Python bindings to the Tree-sitter parsing library";
-    homepage = "https://github.com/tree-sitter/py-tree-sitter";
-    changelog = "https://github.com/tree-sitter/py-tree-sitter/releases/tag/${src.tag}";
-    license = lib.licenses.mit;
-    maintainers = with lib.maintainers; [ fab ];
-  };
+  meta =
+    let
+      # for an -unstable version, we grab the release notes for the last tagged
+      # version it is based upon
+      lastTag = lib.pipe version [
+        lib.splitVersion
+        (lib.take 3)
+        (lib.concatStringsSep ".")
+      ];
+    in
+    {
+      description = "Python bindings to the Tree-sitter parsing library";
+      homepage = "https://github.com/tree-sitter/py-tree-sitter";
+      changelog = "https://github.com/tree-sitter/py-tree-sitter/releases/tag/v${lastTag}";
+      license = lib.licenses.mit;
+      maintainers = with lib.maintainers; [ fab ];
+    };
 }

